jira_metrics/epic_tracking.py

```diff
@@ -223,21 +121,169 @@ def validate_env_variables():
     return env_values
 
 
-def bucket_counts(children: List[Dict[str, Any]]):
-    total = len(children)
-    done = 0
-    inprog = 0
-    other = 0
-    for c in children:
-        status_name = (c["fields"]["status"]["name"] or "").strip().lower()
-        if status_name in DONE_STATUSES:
-            done += 1
-        elif status_name in INPROG_STATUSES:
-            inprog += 1
+def get_quarter_dates(year, quarter):
+    """Get start and end dates for a quarter."""
+    from datetime import timezone
+    
+    quarter_starts = {
+        1: (1, 1), 2: (4, 1), 3: (7, 1), 4: (10, 1)
+    }
+    quarter_ends = {
+        1: (3, 31), 2: (6, 30), 3: (9, 30), 4: (12, 31)
+    }
+    
+    start_month, start_day = quarter_starts[quarter]
+    end_month, end_day = quarter_ends[quarter]
+    
+    start_date = datetime(year, start_month, start_day, tzinfo=timezone.utc)
+    end_date = datetime(year, end_month, end_day, 23, 59, 59, tzinfo=timezone.utc)
+    
+    return start_date, end_date
+
+
+def get_month_dates(year, month):
+    """Get start and end dates for a month."""
+    from datetime import timezone
+    
+    start_date = datetime(year, month, 1, tzinfo=timezone.utc)
```

Comment:
it's UTC, typically we use MST (because that's where I'm at) but UTC I think is fine here. UTC is default in Jira